lib/streamlit/web/server/server.py

```diff
@@ -229,33 +172,12 @@ def __init__(self, main_script_path: str, command_line: Optional[str]):
         _set_tornado_log_levels()
 
         self._main_script_path = main_script_path
-        self._command_line = command_line if command_line is not None else ""
-
-        # Will be set when we start.
-        self._eventloop: Optional[asyncio.AbstractEventLoop] = None
-
-        # Mapping of AppSession.id -> SessionInfo.
-        self._session_info_by_id: Dict[str, SessionInfo] = {}
-
-        self._must_stop = tornado.locks.Event()
-        self._state = State.INITIAL
-        self._message_cache = ForwardMsgCache()
-        self._uploaded_file_mgr = UploadedFileManager()
-        self._uploaded_file_mgr.on_files_updated.connect(self.on_files_updated)
-        self._session_data: Optional[SessionData] = None
```

Comment:
Double-checking -- this is just unused, right?

Reply:
`self._session_data` is unused, yeah (it should not be there in `develop` either).